api/sql/thing_by_id.sql

```diff
@@ -15,6 +15,8 @@ WITH authors_list AS (
 full_thing AS (
   SELECT
     ${table:name}.*,
+    COALESCE(location, '("","","","","","","","","")'::geolocation) AS location,
```

Comment:
Ensures that we always return an empty location rather than null. We do this everywhere else, but I'd missed this one, added for consistency.